### PR TITLE
[VEN-2727] [VEN-2728] [VEN-2729] Add VIPs for core pool, PSR, and NTG for OP sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves:
- VEN-2727: Add VIP for core pool on opsepolia
- VEN-2728: Add VIP for NTG on opsepolia
- VEN-2729: Add VIP for PSR on opsepolia

### CODE REVIEW

File: multisig/proposals/opsepolia/vip-005/index.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { opsepolia } = NETWORK_ADDRESSES;
+
+export const ACM = "0x1652E12C8ABE2f0D84466F0fc1fA4286491B3BC1";
+export const PSR = "0x0F021c29283c47DF8237741dD5a0aA22952aFc88";
+export const NATIVE_TOKEN_GATEWAY_CORE_POOL = "0xfE0AAFd5A4fEB9f6DEC36B0Cb8A972bD0B5149f7";
```
Comment (chechu) [inline multisig/proposals/opsepolia/vip-005/index.ts:8]: According to https://github.com/VenusProtocol/isolated-pools/pull/435/

```suggestion
export const NATIVE_TOKEN_GATEWAY_CORE_POOL = "0x521f59b2670bcc70961FB2edD4F62c3f3B1E7f6b";
```

Comment (Debugger022) [reply multisig/proposals/opsepolia/vip-005/index.ts]: Yes, both addresses are right. I just lost the NTG deployment file while merging this branch with main so needed to deploy the NTG one more time. `0x521f59b2670bcc70961FB2edD4F62c3f3B1E7f6b ` latest one which is mentioned here :https://github.com/VenusProtocol/isolated-pools/pull/435
Anyways, I have updated the address here as well, https://github.com/VenusProtocol/isolated-pools/pull/435

File: multisig/proposals/opsepolia/vip-004/index.ts [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMinLiquidatableCollateral(uint256)", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setActionsPaused(address[],uint256[],bool)", opsepolia.GUARDIAN],
+    },
```
Comment (chechu) [inline multisig/proposals/opsepolia/vip-004/index.ts:61]:
```suggestion
    },
    {
      target: ACM,
      signature: "giveCallPermission(address,string,address)",
      params: [ZERO_ADDRESS, "setForcedLiquidation(address,bool)", opsepolia.GUARDIAN],
    },
```

File: multisig/proposals/opsepolia/vip-004/index.ts [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "addPool(string,address,uint256,uint256,uint256)", opsepolia.GUARDIAN],
```
Comment (chechu) [inline multisig/proposals/opsepolia/vip-004/index.ts:65]:
```suggestion
      params: [opsepolia.POOL_REGISTRY, "addPool(string,address,uint256,uint256,uint256)", opsepolia.GUARDIAN],
```

File: multisig/proposals/opsepolia/vip-004/index.ts [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setPoolName(address,string)", opsepolia.GUARDIAN],
```
Comment (chechu) [inline multisig/proposals/opsepolia/vip-004/index.ts:70]:
```suggestion
      params: [opsepolia.POOL_REGISTRY, "setPoolName(address,string)", opsepolia.GUARDIAN],
```

File: multisig/proposals/opsepolia/vip-004/index.ts [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updatePoolMetadata(address,VenusPoolMetaData)", opsepolia.GUARDIAN],
```
Comment (chechu) [inline multisig/proposals/opsepolia/vip-004/index.ts:75]:
```suggestion
      params: [opsepolia.POOL_REGISTRY, "updatePoolMetadata(address,VenusPoolMetaData)", opsepolia.GUARDIAN],
```

File: multisig/proposals/opsepolia/vip-004/index.ts [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setRewardTokenSpeeds(address[],uint256[],uint256[])", opsepolia.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLastRewardingBlock(address[],uint32[],uint32[])", opsepolia.GUARDIAN],
```
Comment (chechu) [inline multisig/proposals/opsepolia/vip-004/index.ts:105]:
```suggestion
      params: [ZERO_ADDRESS, "setLastRewardingBlockTimestamps(address[],uint256[],uint256[])", opsepolia.GUARDIAN],
```

Comment (Debugger022) [reply multisig/proposals/opsepolia/vip-004/index.ts]: fixed, https://github.com/VenusProtocol/vips/pull/379/commits/87cb376178f1be2313bd520b3152ffe96726f78d

File: multisig/proposals/opsepolia/vip-004/index.ts (hunk 5 above)
Comment (Debugger022) [reply multisig/proposals/opsepolia/vip-004/index.ts]: fixed, https://github.com/VenusProtocol/vips/pull/379/commits/87cb376178f1be2313bd520b3152ffe96726f78d

File: multisig/proposals/opsepolia/vip-004/index.ts (hunk 4 above)
Comment (Debugger022) [reply multisig/proposals/opsepolia/vip-004/index.ts]: fixed, https://github.com/VenusProtocol/vips/pull/379/commits/87cb376178f1be2313bd520b3152ffe96726f78d

File: multisig/proposals/opsepolia/vip-004/index.ts (hunk 3 above)
Comment (Debugger022) [reply multisig/proposals/opsepolia/vip-004/index.ts]: fixed, https://github.com/VenusProtocol/vips/pull/379/commits/87cb376178f1be2313bd520b3152ffe96726f78d

File: multisig/proposals/opsepolia/vip-004/index.ts (hunk 2 above)
Comment (Debugger022) [reply multisig/proposals/opsepolia/vip-004/index.ts:61]: fixed, https://github.com/VenusProtocol/vips/pull/379/commits/87cb376178f1be2313bd520b3152ffe96726f78d

**Review by chechu — APPROVED** [review]

**Comment by Debugger022** [comment]
Multisig tx. to add pool in opsepolia:
- https://sepolia-optimism.etherscan.io/tx/0x4295e4dc08843a89b10fdf905bd01b31f6f11eae38990e6df83dcd61aa068353
- https://sepolia-optimism.etherscan.io/tx/0x0a515189e79957f9ab778f15aa071707f973035c2852cad1f3862ece32131503

**Comment by Debugger022** [comment]
Multisig for PSR and NTG: 
https://sepolia-optimism.etherscan.io/tx/0x8a3803a80667bb649323398c3b045b50358abc9ee595ee148fa39daabf57f54d

**Pull request merged**